node/libs/roles/src/validator/messages/consensus.rs

```diff
@@ -64,6 +64,9 @@ pub enum LeaderSelectionMode {
 
     /// Select pseudo-randomly, based on validators' weights.
     Weighted,
+
+    /// Select based on a sticky assignment to a non-empty list of specific validators.
```

Comment:
Comment is not clear enough on how this mode works.

Reply:
Hm, I thought I sent a new version already 👀

Reply:
Sorry, updated.